vnext/Shared/HermesShim.cpp

```diff
@@ -11,6 +11,8 @@ static decltype(&facebook::hermes::makeHermesRuntime) s_makeHermesRuntime{nullpt
 static decltype(&facebook::hermes::HermesRuntime::enableSamplingProfiler) s_enableSamplingProfiler{nullptr};
 static decltype(&facebook::hermes::HermesRuntime::disableSamplingProfiler) s_disableSamplingProfiler{nullptr};
 static decltype(&facebook::hermes::HermesRuntime::dumpSampledTraceToFile) s_dumpSampledTraceToFile{nullptr};
+static decltype(&facebook::hermes::makeHermesRuntimeWithWER) s_makeHermesRuntimeWithWER{nullptr};
+static decltype(&facebook::hermes::hermesCrashHandler) s_hermesCrashHandler{nullptr};
```

Comment:
can't see the signature here but can we guarantee that this `hermesCrashHandler` function is noexcept / doesn't throw since this is across DLL boundaries (plus it should throw in a WER handler)

Reply:
It's not marked noexcept, but HermesRuntimeHolder::crashHandler (the wrapper) is.

I think we'll address the DLL boundary issue further with the ABI safety work.